analyzers/src/SonarAnalyzer.CSharp/LiveVariableAnalysis/CSharp/CSharpLiveVariableAnalysis.cs

```diff
@@ -178,6 +182,16 @@ private void ProcessIdentifier(IdentifierNameSyntax identifier, State state)
             }
         }
 
+        private void ProcessGenericName(SyntaxNode genericName, State state)
+        {
+            if (genericName.Parent.IsKind(SyntaxKind.InvocationExpression)
```

Comment:
This is the same logic as in the `ProcessIdentifier(..)` above. We just don't need to care about the `if (IsLocalScoped(..` here.